easybuild/easyconfigs/g/GMP/GMP-6.0.0a-intel-2015a.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'GMP'
+version = '6.0.0a'
+
+homepage = 'http://gmplib.org/'
+description = """GMP is a free library for arbitrary precision arithmetic, 
+operating on signed integers, rational numbers, and floating point numbers. """
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'optarch': True, 'pic': True, 'precise': True}
```

Comment:
Our original build was with iccifort 2015.3.187, just tested with 2015a and that also succeeds